front/src/applications/operationalStudies/components/Scenario/AddOrEditScenarioModal.tsx

```diff
@@ -155,11 +189,6 @@ export default function AddOrEditScenarioModal({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [infraID]);
 
-  useEffect(() => {
-    createElectricalProfileSetOptions();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
-
   return electricalProfileSetOptions ? (
```

Comment:
Can you move the condition ? We need to test if electricalProfileSetOptions is empty or not to display only the dropdown to select the electrical profile
```suggestion
  return (
```

Reply:
Done !!!